nodeadm/internal/system/networking.go

```diff
@@ -89,11 +96,79 @@ func (a *networkingAspect) ensureEKSNetworkConfiguration(cfg *api.NodeConfig) er
 	return nil
 }
 
+func (a *networkingAspect) ensureMulticardNetworkConfiguration(cfg *api.NodeConfig) error {
+	var networkRestartRequired bool
+	routeTableId := 1001
+	routePriority := 32765
+
+	for _, card := range cfg.Status.Instance.NetworkCards {
+		if card.CardIndex == 0 {
+			continue
+		}
+
+		networkInterfaceConfName := fmt.Sprintf("80-card%d.network", card.CardIndex)
```

Comment:
IIRC, it should be possible to have multiple ENIs per network card, have we tested this behavior?
seems it won't work if you have multiple ENI on a network card

Reply:
I see your point. The problem we face is just the name of the file right? I can add the deviceIndex in the name: `80-card%d-%d.network`, of we can use the mac address too. Do you have any preference?

Reply:
i'd like to avoid couple to cardIndex&deviceIndex(it's complicated, see https://github.com/amazonlinux/amazon-ec2-net-utils/blob/e01f53f278eeb13bbdc856da921584944c825286/lib/lib.sh#L344)

I prefer to use `70-<eni-xxxx>.network` where `eni-xxxx` is the eni id, [you can get it from ec2Metadata](https://github.com/amazonlinux/amazon-ec2-net-utils/blob/e01f53f278eeb13bbdc856da921584944c825286/lib/lib.sh#L334)